src/community/oseo/oseo-stac/src/test/java/org/geoserver/ogcapi/stac/ItemsTest.java

```diff
@@ -145,7 +145,7 @@ public void testSentinelItemsHTML() throws Exception {
     }
 
     private void testSentinel2SampleHTML(Elements elements) {
-        assertTextContains(elements, "[data-tid='gbounds']", "-119,174, 33,333, -117,969, 34,338.");
+        assertTextContains(elements, "[data-tid='gbounds']", "-119.174, 33.333, -117.969, 34.338.");
```

Comment:
Uh oh, another locale issue. Same as above, make the test Locale independent by setting a default locale at the begining of the test (possibly in the upper-most test class of this module).

Reply:
Fixed by setting Locale in parent.